stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py

```diff
@@ -84,6 +99,6 @@ def _prepare_data_from_move_line(self, move_line):
     def print_labels(self):
         print_move = self.product_print_moves.filtered(lambda p: p.label_qty > 0)
         if print_move:
-            return self.env.ref(
-                "stock_picking_product_barcode_report.action_label_barcode_report"
-            ).report_action(self.product_print_moves)
+            return self.env.ref(self.barcode_report.xml_id).report_action(
```

Comment:
You no you need this? Would this work?
```suggestion
            return self.barcode_report.report_action(
```